ServerCore/Pages/Teams/Details.cshtml.cs

```diff
@@ -146,6 +148,10 @@ orderby lunch.ID
                 SoftMaxLunches = (int)Math.Ceiling(possibleInPersonMembers / (double)PlayersPerLunch);
             }
 
+            // Get team room
+            Room teamRoom = _context.Room.Where(r => r.TeamID == teamId).FirstOrDefault();
+            TeamRoom = $"{teamRoom.Building}/{teamRoom.Number}({teamRoom.Capacity})";
```

Comment:
This will throw if the team isn't assigned a room

Reply:
Added null check.